src/bundle/Resources/public/js/scripts/admin.location.tab.js

```diff
@@ -1,11 +1,12 @@
-(function () {
-    var url = document.location.toString();
+(function (global, doc, jQuery) {
+    const $ = jQuery;
+    var url = doc.location.toString();
     if (url.match('#')) {
```

Comment:
`window.location.hash` returns `''` (empty string) if there's no hash and a string like `'#4'`if there's any.
So no need to stringify `window.location` and no need to do the match check.